metamath.tex

```diff
@@ -7374,7 +7374,7 @@ \chapter{The Metamath Language}
 
 The next section contains the complete specification of the Metamath
 language.\footnote{The current version of the Metamath program
-(0.07.30)\index{Metamath!limitations of version 0.07.30} implements an
```

Comment:
Let's take out this entire footnote.  (metamath.exe should implement it exactly unless there is a bug.)

Reply:
Done, it'll be in the next commit on this branch.